src/go/pdwfs.go

```diff
@@ -336,8 +341,9 @@ func Read(fd int, buf []byte) int {
 
 	n, err := (*file).Read(buf)
 	if err != nil && err != io.EOF {
-		panic(err)
-	}
+		// panic(err)
```

Comment:
same comment as previous one

Reply:
Same as before with write only error this time.